config/opal_check_package.m4

```diff
@@ -134,6 +134,8 @@ AC_DEFUN([_OPAL_CHECK_PACKAGE_LIB], [
                   test "$ac_cv_search_$3" != "none required"],
                  [$1_LIBS="$ac_cv_search_$3 $4"],
                  [$1_LIBS="$4"])
+            # Check libnl version if any
+            OPAL_CHECK_LIBNL_VERSION([$2])
```

Comment:
...and also $3?  I.e., if you don't link in a symbol from the library in the LIBNL test, then it might not actually pull in the $2 library...